src/hooks/useAllTickets.ts

```diff
@@ -4,9 +4,15 @@ import useSWR from 'swr';
 export default function useAllTickets() {
   const {
     data, isLoading, isValidating,
-  } = useSWR('dataservice/alltickets', async () => DataService.getServiceRequestSummary());
+  } = useSWR(
+    'dataservice/alltickets',
+    async () => DataService.getServiceRequestSummary(),
+    { refreshInterval: 10000 },
+  );
 
   return {
-    data: data ?? [], isLoading, isValidating,
+    data,
+    isLoading: !data && isLoading,
```

Comment:
I don't know if this is the right way to do this. I found that without this there was a lot of jitter on the webpage. It seems like this removes it. My assumption is that the isLoading was being set true and made the whole table show the isolating state, even when the data did not change.